src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -3089,31 +3089,29 @@ public static Vector128<sbyte> Shuffle(Vector128<sbyte> vector, Vector128<sbyte>
         /// <param name="vector">The input vector from which values are selected.</param>
         /// <param name="indices">The per-element indices used to select a value from <paramref name="vector" />.</param>
         /// <returns>A new vector containing the values from <paramref name="vector" /> selected by the given <paramref name="indices" />.</returns>
-        /// <remarks>Unlike Shuffle, this method delegates to the underlying hardware intrinsic without ensuring that <paramref name="indices" /> are normalized to [0, 15].
-        /// On hardware with <see cref="Ssse3" /> support, indices are treated as modulo 16, and if the high bit is set, the result will be set to 0 for that element.
-        /// On hardware with <see cref="AdvSimd.Arm64" /> or <see cref="PackedSimd" /> support, this method behaves the same as Shuffle.</remarks>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        /// <remarks>Unlike Shuffle, this method delegates to the underlying hardware intrinsic without ensuring that <paramref name="indices"/> are normalized to [0, 15].</remarks>
+        [Intrinsic]
         [CompExactlyDependsOn(typeof(Ssse3))]
-        [CompExactlyDependsOn(typeof(AdvSimd))]
```

Comment:
@tannergooding for when you review this PR, see this commit for all the `[CompExactlyDependsOn]`s we can remove technically https://github.com/dotnet/runtime/commit/7b129a26b5e08aed04ae872bea25e995726a6f30. I've added them back (except the one that seems completely useless, `AdvSimd`, as we don't emit anything special for V128 shuffle without `AdvSimd.Arm64`), as I'm not certain if we do want to remove them (since it will force rejitting the code when `AdvSimd` is present for example, which is probably ideal in the case of these APIs). Let me know whether I should remove them or leave like it is now, or something else.